### PR TITLE
e2e tests - Shortcuts and Workbench

### PR DESCRIPTION
**Pull request by tanyatrayanava**

### CODE REVIEW

File: tests/e2e/tests/critical-path/workbench/command-results.e2e.ts [hunk 1]
```diff
@@ -75,3 +77,21 @@ test('Verify that user can see the results found in the table view by default fo
         await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssTableViewTypeOption).visible).ok(`The table view is selected by default for command ${command}`);
     }
 });
+test.skip('Verify that user can switches between views and see results according to the view rules in Workbench in results', async t => {
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/workbench/command-results.e2e.ts:80]: Please add a reason for the skip

File: tests/e2e/tests/critical-path/workbench/command-results.e2e.ts [hunk 2]
```diff
@@ -75,3 +77,21 @@ test('Verify that user can see the results found in the table view by default fo
...
+    const commands = [
+        'hset doc:10 title "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud" url "redis.io" author "Test" rate "undefined" review "0" comment "Test comment"',
+        `FT.CREATE ${indexName} ON HASH PREFIX 1 doc: SCHEMA title TEXT WEIGHT 5.0 body TEXT url TEXT author TEXT rate TEXT review TEXT comment TEXT`,
+        `FT.SEARCH ${indexName} * limit 0 10000`
+    ];
+    //Send commands and check table view is default for Search command
+    for(let command of commands) {
+        await workbenchPage.sendCommandInWorkbench(command);
+    }
+    await t.expect(await workbenchPage.queryCardContainer.nth(0).find(workbenchPage.cssTableViewTypeOption).visible).ok(`The table view is selected by default for command FT.SEARCH`);
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/workbench/command-results.e2e.ts:90]: Please use with `'` and not `

Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/workbench/command-results.e2e.ts]: done

File: tests/e2e/tests/critical-path/workbench/command-results.e2e.ts (hunk 1 above)
Comment (tanyatrayanava) [reply tests/e2e/tests/critical-path/workbench/command-results.e2e.ts:81]: done

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**